titan/probabilities.py

```diff
@@ -5,28 +5,20 @@
 
 
 @utils.memo
-def adherence_prob(adherence: int) -> float:
+def adherent_prob(adherent: bool) -> float:
     """
-    Mapping from HAART adherence levels to probabilities.
+    Mapping from HAART q levels to probabilities.
```

Comment:
what are `q levels`?  Also, wondering if this should still be in our prob file since it's not so complex anymore?

Reply:
Adherence of course! (oops)

Reply:
This is probably a to-do to parameterize it, so I'm going to leave it for now.